extension/oidcauthextension/config.go

```diff
@@ -20,7 +20,7 @@ import "go.opentelemetry.io/collector/config"
 type Config struct {
 	config.ExtensionSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
-	// The attribute (header name) to look for auth data. Optional, default value: "authorization".
+	// The attribute (header name) to look for auth data. Optional, default value: "Authorization".
```

Comment:
This would break gRPC. We should have a logic somewhere where we'd traverse the incoming http request map looking for this header in a case-insensitive manner.